### PR TITLE
Feature/wf 623 social media

### PR DESCRIPTION
**Pull request by geroldboehler**
## Pull request
Adds c-social-media component.
 
### Ticket
https://winterhalter-fenner.atlassian.net/browse/WF-623
 
### Browser testing
http://localhost:8080/styleguide/layout
 
### Checklist
- [x] Added JSDoc and styleguide demo
- [x] Tested all links in project relevant browsers
- [x] Tested all links in different screen sizes
- [x] Did run automated tests and linters
- [x] Did assign ticket
- [x] Double checked target branch
 
## Review/Test checklist
- [x] Did run automated tests and linters
- [x] Did review code and documentation
- [x] Tested all links in project relevant browsers
- [x] Tested all links in different screen sizes
- [x] Re-assign ticket to developer

### CODE REVIEW

File: app/components/c-social-media.vue [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    }
+
+    &__follow-us,
+    &__link {
+      flex: 0 1 auto;
+    }
+
+    &__link {
+      display: block;
+      margin-left: $spacing--10;
```
Comment (claudio147) [inline app/components/c-social-media.vue]: - [ ] In the design the vertical space between the icons is `20px`

File: app/components/c-footer.vue [hunk 2]
```diff
@@ -20,9 +24,13 @@
 </template>
 
 <script>
+  import cSocialMedia from '@/components/c-social-media';
+
   export default {
     name: 'c-footer',
```
Comment (claudio147) [inline app/components/c-footer.vue]: I think, according to the design the spacing on the right / left should be more than 20px

**Comment by claudio147** [comment]
Ob Android (via Browserstack) the icon borders aren't really visible. Please check this on a real device.
bildschirmfoto 2018-05-11 um 15 42 30

File: app/components/c-social-media.vue [hunk 3]
```diff
@@ -0,0 +1,88 @@
+<template>
+  <div :class="b()">
+    <div :class="b('follow-us')">{{ $t('c-social-media.followUs') }}</div>
+    <a :class="b('link')" :href="$t('c-social-media.facebookUrl')" target="_blank">
```
[the remark this answers is not in the document]
Comment (geroldboehler) [reply app/components/c-social-media.vue]: Links are hardcoded, yes

File: app/components/c-social-media.vue [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+      <e-icon :class="b('icon')" inline icon="i-facebook"/>
```
[the remark this answers is not in the document]
Comment (geroldboehler) [reply app/components/c-social-media.vue]: Sorry don't understand that one

File: app/components/c-social-media.vue [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    }
+
+    &__link {
+      display: block;
+      margin-left: $spacing--10;
+    }
+
+    &__icon {
+      width: $spacing--30;
+      height: $spacing--30;
```
[the remark this answers is not in the document]
Comment (geroldboehler) [reply app/components/c-social-media.vue]: I also think thats a design change and leave it for now. I will ask christoph to make sure.

File: app/components/c-footer.vue (hunk 2 above)
Comment (geroldboehler) [reply app/components/c-footer.vue]: Done

**Comment by geroldboehler** [comment]
I don't have an android device at hand, but i did use browserstack and used the phone in portrait mode and scrolled. there you can see that the lines interpolate, depending on the y position of the site. Also when adding a 2px border, you see both borders, however one with 1px and one with 2px :)

**Comment by claudio147** [comment]
Checked the icons on a android device and looks fine.

**Pull request merged**